components/EyeDropper/src/index.svelte

```diff
@@ -29,10 +30,8 @@
 	}
 
 	onMount(initEyeDropper);
-
-	$: cnames = clsx(cls);
 </script>
 
-<div class="k-eyedropper--base {cnames}" {...attrs}>
+<div class={clsx(cls, 'k-eyedropper--base')} {...attrs}>
```

Comment:
I think the cls passed in should override the base style.
```svelte
<div class={clsx('k-eyedropper--base', cls)} {...attrs}>...</div>
```

Reply:
yes, cls should always be last

Reply:
The class defined last in the CSS is what wins in these cases. The order on the element doesn't matter.